inc/item_disk.class.php

```diff
@@ -337,6 +342,27 @@ static function showForItem(CommonDBTM $item, $withtemplate = 0) {
             }
             Html::displayProgressBar('100', $percent, ['simple'       => true,
                                                             'forcepadding' => false]);
+            echo "</td>";
+            echo "<td class=\"center\">";
+
+            if ($data['encryption_status'] != self::ENCRYPTION_STATUS_NO) {
+               $encryptionID = "disk-{$data["id"]}-encryption";
+               $encryptionTooltip = "<strong>" . __('Partial encryption') . "</strong> : " .
+                  Dropdown::getYesNo($data['encryption_status'] == self::ENCRYPTION_STATUS_PARTIALLY) .
+                  "<br/>" .
+                  "<strong>" . __('Encryption tool') . "</strong> : " . $data['encryption_tool'] .
+                  "</br>" .
+                  "<strong>" . __('Encryption algorithm') . "</strong> : " .
+                  $data['encryption_algorithm'] . "</br>" .
+                  "<strong>" . __('Encryption type') . "</strong> : " . $data['encryption_type'] .
+                  "</br>";
+
+               echo "<i id=\"$encryptionID\" class=\"fas fa-lock\"></i> &nbsp;";
+               echo Html::showTooltip($encryptionTooltip, [
```

Comment:
`echo` should be removed `showTooltip` does already it by default.
```suggestion
               Html::showTooltip($encryptionTooltip, [
```